drivers/CMakeLists.txt

```diff
@@ -46,22 +47,23 @@ target_sources(mbed-os
         source/UnbufferedSerial.cpp
         source/Watchdog.cpp
 
-        source/usb/AsyncOp.cpp
-        source/usb/ByteBuffer.cpp
-        source/usb/EndpointResolver.cpp
-        source/usb/LinkedListBase.cpp
-        source/usb/OperationListBase.cpp
-        source/usb/PolledQueue.cpp
-        source/usb/TaskBase.cpp
-        source/usb/USBAudio.cpp
-        source/usb/USBCDC.cpp
-        source/usb/USBCDC_ECM.cpp
-        source/usb/USBDevice.cpp
-        source/usb/USBHID.cpp
-        source/usb/USBKeyboard.cpp
-        source/usb/USBMIDI.cpp
-        source/usb/USBMSD.cpp
-        source/usb/USBMouse.cpp
-        source/usb/USBMouseKeyboard.cpp
-        source/usb/USBSerial.cpp
+        #source/usb/AsyncOp.cpp
```

Comment:
this is PITA, drivers are not pure drivers. I need to raise this with core team. device key is neither. They should not be enabled like this otherwise we pull in storage always + mbedtls and others :(

Reply:
fixed